docs/Prerequisites.adoc

```diff
@@ -3,14 +3,10 @@
 === Installing Protocol Buffers Compiler
 
 ==== For Windows
-Install the binary from https://github.com/google/protobuf/releases/download/v2.4.1/protoc-2.4.1-win32.zip and then set the `/path/to/protoc/parent` on your PATH variable.
+For Windows, the easiest way to install the compiler is to install the binary from https://github.com/google/protobuf/releases/download/v2.6.1/protoc-2.6.1-win32.zip and then set the `/path/to/protoc/parent` on your PATH variable.
```

Comment:
Hmm, one is the header title. Is the following "For Windows" redundant, should I remove it? I thought it would make things even more clear to the reader, but maybe not..